src/features/problems/form/edit-problem-form.tsx

```diff
@@ -147,9 +157,14 @@ const EditProblemForm: React.FC<OwnProps> = ({ id, problem }) => {
             <div className="sticky top-0">
               <h2 className="min-w-[200px] text-lg font-medium">Problem details</h2>
             </div>
-            <div className="flex w-full flex-col gap-4">
+            <div className="flex w-full min-w-0 flex-col gap-4">
```

Comment:
do `MarkdownEditor` or `TextField` have their own `min-w` property, making it necessary to use `min-w-0`?

Reply:
Same as above (overflow issue).